src/translations/en.json

```
@@ -1049,7 +1049,7 @@
               "introduction": "Some parts of Home Assistant can reload without requiring a restart. Hitting reload will unload their current YAML configuration and load the new one.",
               "reload": "Reload {domain}",
               "core": "Reload location & customizations",
-              "group": "Reload groups, group entities, and notify services",
+              "group": "Reload groups, group entities and notify services",
```

```suggestion
              "group": "Reload groups, group entities, and group notify services",
```
Wouldn't that make it even clearer?

---

Works for me :)

---

Then I would propose we do the same for "rest" to be consistent.

---

If I didn't miss any, there are three other strings that mention notify services.
On one hand `telegram` and `smtp` are probably fine:
https://github.com/home-assistant/frontend/blob/a8d01346e893bb7d16f8fe442fcfd80ec205fb59/src/translations/en.json#L1077-L1078

`rest` on the other could be changed:
https://github.com/home-assistant/frontend/blob/a8d01346e893bb7d16f8fe442fcfd80ec205fb59/src/translations/en.json#L1065

Although to be fair I don't think it's as bad as with `group` since it IMO clearly applies to rest.